easybuild/easyconfigs/t/Tk/Tk-8.6.7-iomkl-2018a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.7'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
+ building a graphical user interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+patches = ['Tk-8.6.4_different-prefix-with-tcl.patch']
+checksums = [
+    '061de2a354f9b7c7d04de3984c90c9bc6dd3a1b8377bb45509f1ad8a8d6337aa',  # tk8.6.7-src.tar.gz
+    '7a6daa8349393af3d340e774aebf07c7410c51e01bc654ceb3679877063b961d',  # Tk-8.6.4_different-prefix-with-tcl.patch
+]
+
+dependencies = [
+    ('X11', '20171023'),
```

Comment:
bump to `20180131`?

Reply:
Hi @boegel - so would you like me to split this one up, or continue with it?

Reply:
I would say splitting it up makes more sense, this is going to remain a PITA otherwise...

Maybe start with a PR for `libGLU` + deps (with updated versions)? A separate PR for `FFTW` also makes sense imho.

You can keep this one open and sync it with `develop` to trim it down to something more manageable...

Reply:
OK (I've now said this in two places, but better to be clear...). I'll split it up.